src/coreclr/jit/lowerxarch.cpp

```diff
@@ -2592,6 +2592,17 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
             return LowerHWIntrinsicTernaryLogic(node);
         }
 
+        case NI_PCLMULQDQ_CarrylessMultiply:
```

Comment:
This shouldn't be needed, instead we should be customizing how the intrinsic is built in `lsraxarch.cpp` to give a different mask based on if `VCPLMULQDQ+EVEX` is defined or not.